multimedia/mat2/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                mat2
+version             0.13.3
+categories          multimedia
+license             LGPL-3
+supported_archs     noarch
+platforms           {darwin any}
+maintainers         {@akierig fastmail.de:akierig} \
+                    openmaintainer
+
+description         metadata removal tool
+long_description    mat2 is a metadata removal tool, supporting a wide range \
+                    of commonly used file formats, written in python3.
+homepage            https://0xacab.org/jvoisin/mat2
+master_sites        https://0xacab.org/jvoisin/mat2/-/archive/${version}/
+
+checksums           rmd160  69de0de9284552f001bd5f4aa2210b021a42eca9 \
+                    sha256  e1bb0161fa1c2f2adb1b933761f9569534309e90209568f96aca5fc8a74f511a \
+                    size    11782599
+
+python.versions     39 310
```

Comment:
it looks like this is intended as a Python program, not really to be imported as a package? Additionally, since the port is not in the "python" category it's not helpful to use `python.versions` as no subports will be created anyway. So you will want to pick the current default Python versions, which is 311, and instead to `python.default_version 311`.

Reply:
`py311-gobject3` can be dealt with in a separate PR, following which the python version for mat2 can always be bumped. I don't think that should hold up merging this PR. 

I agree with @reneeotten though that `python.versions` doesn't make sense here since there are no subports (i.e. pyVER-...). It should then be `python.default_version 310`.

Reply:
> It can be used as either a library or just a CLI program, I use it as the later. 

If you want to be able to use it as both it should be in the "python" category so that the pyXY subports are generated. In that case any programs/scripts will be suffixed by "-3.X". 

> the problem here is that there's no `py311-gobject3` which of course ends with the build failing. Not sure if @dbevans is planning to bump the version soon or maybe it doesn't work with 3.11?

Sorry I didn't pay attention to whether dependencies already had a py311 subport. So sure, updating to use Python 3.11 can be done later.